internal/predicate/velerobackup_predicate.go

```diff
@@ -36,30 +36,23 @@ type VeleroBackupPredicate struct {
 	Logger              logr.Logger
 }
 
-// TODO try to remove calls to get logger functions, try to initialize it
```

Comment:
why TODO was removed?

Reply:
I think this is a similar problem in OADP https://github.com/openshift/oadp-operator/issues/1127 and believe the implementation can be improved

Reply:
Yes it can be improved. We can track it in separate issue. At the moment I think its ok.

Reply:
Yes, it can be resolved another moment, I agree

BUT it is still not resolved. Either delete TODO by creating issue, or do not delete it from the code

Reply:
Added TODO back